<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="BDC6-DX-840RX52" ignored="0" name="Default test" tests="1" failures="0" timestamp="2020-01-24T12:12:37 IST" time="0.202" errors="0">
  <testcase name="@BeforeTest startReport" time="0.113" classname="ExtentsReport.ExtentReport1">
    <failure type="java.lang.Error" message="Unresolved compilation problem: 
	VERSION_2_3_29 cannot be resolved or is not a field
">
      <![CDATA[java.lang.Error: Unresolved compilation problem: 
VERSION_2_3_29 cannot be resolved or is not a field

at com.aventstack.extentreports.reporter.BasicFileReporter.start(BasicFileReporter.java:98)
at com.aventstack.extentreports.ReportObservable.register(ReportObservable.java:191)
at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948)
at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:580)
at com.aventstack.extentreports.ExtentReports.attachReporter(ExtentReports.java:82)
at ExtentsReport.ExtentReport1.startReport(ExtentReport1.java:36)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:623)
at org.testng.TestRunner.beforeRun(TestRunner.java:613)
at org.testng.TestRunner.run(TestRunner.java:584)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest startReport -->
  <testcase name="@AfterMethod getResult" time="-1.579848157559E9" classname="ExtentsReport.ExtentReport1">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterTest endReport" time="-1.579848157571E9" classname="ExtentsReport.ExtentReport1">
    <skipped/>
  </testcase> <!-- @AfterTest endReport -->
  <testcase name="passTest" time="0.0" classname="ExtentsReport.ExtentReport1">
    <skipped/>
  </testcase> <!-- passTest -->
</testsuite> <!-- Default test -->
